python-numpy-basics/Classes_and_Objects.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>use <a href="https://www.markdownguide.org/basic-syntax/#ordered-lists" target="_blank">ordered list</a> for content under Headers</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
yes, i have changed it